src/cpu/x64/injectors/jit_uni_binary_injector.cpp

```diff
@@ -58,6 +58,7 @@ bool is_data_supported(cpu_isa_t isa, data_type_t data_type) {
                     || is_superset(isa, avx2_vnni_2);
         case data_type::f8_e5m2:
         case data_type::f8_e4m3: return is_superset(isa, avx512_core_fp16);
+        case data_type::f64: return false;
         default: return true;
```

Comment:
Let's add rest data types supported by the library here returning `false`, as `f64`, add an `assert` under `default` for 'unsupported data type', and return `false` in `default` instead to prevent searching for issues in the future.